store/generated_key/memstore/memstore.go

```diff
@@ -222,16 +222,6 @@ func (e *MemStore) Verify(_, _ []byte) error {
 	return nil
 }
 
-// Stats ... returns the current usage metrics of the in-memory key-value data store.
-func (e *MemStore) Stats() *store.Stats {
-	e.RLock()
-	defer e.RUnlock()
-	return &store.Stats{
-		Entries: len(e.store),
-		Reads:   e.reads,
-	}
-}
```

Comment:
Why are you removing this? Was this just not used anymore?

Reply:
nope - legacy at this point